src/hotspot/cpu/riscv/vtableStubs_riscv.cpp

```diff
@@ -89,8 +89,8 @@ VtableStub* VtableStubs::create_vtable_stub(int vtable_index) {
     start_pc = __ pc();
 
     // check offset vs vtable length
-    __ lwu(t0, Address(t2, Klass::vtable_length_offset()));
```

Comment:
There is an extra `lwu` -> `lw` which doesn't happen anywhere else.

Reply:
Thanks for your review. fixed.